src/account.ts

```diff
@@ -324,4 +335,25 @@ export class Account {
         });
         return result;
     }
+
+    /**
+     * Returns calculated account balance
+     * @returns {Promise<AccountBalance>}
+     */
+    private async getAccountBalance(): Promise<AccountBalance> {
```

Comment:
any reason to make this private method?

Reply:
I was planning to add it to the state to avoid calling `getAccountBalance` in Wallet separately.